conf/uppmax.config

```diff
@@ -7,6 +7,7 @@ params {
 
 singularity {
   enabled = true
+  cacheDir = '/sw/data/uppnex/ToolBox/nf-core'
```

Comment:
Will this break stuff if people don't have write permissions and the image is not already there?

Reply:
The plan is to have all nf-core pipelines singularity images there whenever we have releases at some point